src/Layers/xrRenderGL/glSH_RT.cpp

```diff
@@ -22,14 +22,16 @@ CRT::~CRT()
 
 bool CRT::used_as_depth() const
 {
+    if (fmt == MAKEFOURCC('D', 'F', '2', '4'))
+        return true;
+
     switch (fmt)
     {
     case D3DFMT_D16:
     case D3DFMT_D16_LOCKABLE:
     case D3DFMT_D15S1:
     case D3DFMT_D24X8:
     case D3DFMT_D24S8:
-    case MAKEFOURCC('D', 'F', '2', '4'):
```

Comment:
What's the warning that complains about this?

Reply:
`error: case value ‘875710020’ not in enumerated type ‘const D3DFORMAT’ [-Werror=switch]`